Utilities/File.cpp

```diff
@@ -1821,6 +1822,28 @@ bool fs::remove_all(const std::string& path, bool remove_root)
 	return true;
 }
 
+std::string fs::resolve_path(std::string_view path)
+{
+	std::error_code ec{};
+	const auto result = std::filesystem::weakly_canonical(std::filesystem::path(path), ec);
+
+	if (ec)
+	{
+		g_tls_error = error::inval;
+		return {};
+	}
+
+	std::string res_str = result.string();
+
+	if constexpr (decltype(result)::preferred_separator != '/')
+	{
+		// For Windows, which the preferred separator of it is '\\'
+		std::replace(res_str.begin(), res_str.end(), ::narrow<char>(decltype(result)::preferred_separator), '/');
```

Comment:
It looks like it's going to break network locations which can only start from \\

Reply:
Fixed.